operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -317,9 +317,10 @@ local utils = import 'mixin-utils/utils.libsonnet';
           alert: $.alertName('StoreGatewayTooManyFailedOperations'),
           'for': '5m',
           expr: |||
-            sum by(%(alert_aggregation_labels)s, operation) (rate(thanos_objstore_bucket_operation_failures_total{component="store-gateway"}[%(range_interval)s])) > 0
+            sum by(%(alert_aggregation_labels)s, %(per_instance_label)s, operation) (rate(thanos_objstore_bucket_operation_failures_total{component="store-gateway"}[%(range_interval)s])) > 0
```

Comment:
this would create one alert per store-gateway zone, which will make this alert very noisy. I'm also inclined to change the message instead of the query